packages/ocaml-option-nnpchecker/ocaml-option-nnpchecker.1/opam

```diff
@@ -0,0 +1,9 @@
+opam-version: "2.0"
+synopsis: "Set OCaml to be compiled with --enable-naked-pointers-checker"
+depends: [
+  "ocaml-variants" {post & >= "4.12.0~"}
+]
+available: arch = "x86_64" & (os = "linux" | os = "macos" | os = "openbsd" | os = "freebsd")
```

Comment:
is the option really specific to those platforms and architecture?

Reply:
It is indeed, see https://github.com/ocaml/ocaml/blob/trunk/configure.ac#L1727-L1735. Technically msvc64 and solaris are also included, but I know we don't have an encoding for msvc64 and I haven't got access to Solaris to see what `uname -s` does (I think it's `SunOS`, but I have no way to check!)

Reply:
Perhaps @mtelvers still has a Solaris box gathering dust somewhere? 😉

Reply:
@dra27 I have recovered my SparcStation 20 from deep storage and even found a Sun 13W3-M to VGA adapter but sadly none of my monitors accept the resolution/timing.

Reply:
@dra27 It's SunOS -- the Wikipedia page has a convenient list: https://en.wikipedia.org/wiki/Uname

Reply:
Ta, @avsm! A handy list (if less fun than persuading Mark to try building OCaml on a 1994 Sun Workstation 😉). I've pushed an amended version with `os = "sunos"` too (opam normalises with `lowercase_ascii`)